pupil_src/shared_modules/background_helper.py

```diff
@@ -64,6 +66,10 @@ def _wrapper(self, pipe, _should_terminate_flag, generator, *args, **kwargs):
             pipe.close()
             logger.debug("Exiting _wrapper")
 
+    @abc.abstract_method
```

Comment:
For this to have an effect your also have to add `metaclass=abc.ABCMeta` to the class definition, i.e. 
`class Task_Proxy(object, metaclass=abc.ABCMeta):`

Reply:
I usually prefer way (1) but I see the annoyance.